climada/hazard/centroids/centr.py

```diff
@@ -694,14 +693,31 @@ def from_csv(cls, file_path):
             on_land = None
         return cls(lat=df['lat'], lon=df['lon'], region_id=region_id, on_land=on_land, crs=crs)
 
+    def write_csv(self, file_path):
+        """Save centroids as CSV file
+
+        Parameters
+        ----------
+        file_path : str
+            absolute or relative file path and name to write to
+        """
+        LOGGER.info('Writing %s', file_path)
+        df = pd.DataFrame(self.gdf)
+        df['lon'] = self.gdf['geometry'].x
+        df['lat'] = self.gdf['geometry'].y
+        df = df.drop(['geometry'], axis=1)
+        crs = CRS.from_user_input(self.crs).to_wkt()
+        df['crs'] = crs
```

Comment:
Since you use this code bit at least twice, you could consider making it a separate method/function that is then called. This avoids the duplication of code.

Reply:
And define that method/function in the test file? Because I don't think there is a general use case for it. Also, from an external point of view, isn't it easier for someone to understand what the test does if they see these lines instead of having to search for the function?

Reply:
In general, one wants to avoid duplicating code. In general, if the function has a good name, there is no need to search for the function. One understands quicker and easier what is being done with a good function name. Also, think for instance that in the future one wants to write a `to_json` method. If the suggested method exists, this becomes easy. If not, then one has to again copy a lot of code.

What do you mean by 'And define that method/function in the test file? '? Do you mean writing a unit test for it?

Reply:
Okay I understand! What I meant is where should I define the function? E.g. in the first test I use it or just on top of the testing file?

Reply:
Not in the test file, but in the `centr.py` file. This is where you use the code bits right?

Reply:
Ah yes sorry I misread and thought this was referring to some code I wrote in the tests, so that's where my confusion came from